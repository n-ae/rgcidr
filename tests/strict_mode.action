-x 192.168.1.1 tests/mixed.given
