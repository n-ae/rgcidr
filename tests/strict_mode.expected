192.168.1.1 is a private IP
